### PR TITLE
Make logger categories match class names

### PR DESCRIPTION
**Pull request by Xiaohongt**
#32

### CODE REVIEW

File: samples/SignalRSample.Web/Startup.cs [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
 
 namespace SignalRSample.Web
 {
     public class Startup
     {
-        public void Configure(IApplicationBuilder app)
+        public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
         {
+#if ASPNET50
+            var serilog = new LoggerConfiguration()
```
Comment (davidfowl) [inline samples/SignalRSample.Web/Startup.cs]: Use the console logger instead of serilog

**Comment by davidfowl** [comment]
Are these all of the categories? Looks tiny. Make sure we didn't miss any.

**Comment by davidfowl** [comment]
:shipit:

**Comment by halter73** [comment]
This has been pushed to dev already. I noticed it is sill using serilog in the sample. Do we still want to change this @davidfowl?

**Comment by Xiaohongt** [comment]
Since SignalR log is huge, it is not easy to read the logs on console, I have talked to @davidfowl about it.

**Pull request closed without merging**